awx/main/access.py

```diff
@@ -1075,8 +1081,8 @@ def filtered_queryset(self):
     @check_superuser
     def can_add(self, data):
         if not data:  # So the browseable API will work
-            return Organization.accessible_objects(self.user, 'admin_role').exists()
-        return self.check_related('organization', Organization, data, mandatory=True)
+            return Organization.accessible_objects(self.user, 'project_admin_role').exists()
+        return self.check_related('organization', Organization, data, role_field='project_admin_role', mandatory=True)
 
     @check_superuser
     def can_change(self, obj, data):
```

Comment:
In order to be consistent with the changes you made to other resources, you should change this `can_change` method

`if not self.check_related('organization', Organization, data, obj=obj):` -> `if not self.check_related('organization', Organization, data, role_field='project_admin_role`, obj=obj):`

or whatever the exact syntax is.

This change is permitting users who hold project_admin_role in 2 organizations to transfer a project from one to another. This is opposed to the current state, which requires that they hold admin_role to both organization.

Minor nit, yes, but you made a similar change for other types, so either go all-in for all of them or none of them.